src/code/dialogs/zoneDialog.js

```diff
@@ -222,13 +179,18 @@ const ZonedrawHandler = L.Handler.extend({
       return;
     }
     L.Handler.prototype.enable.call(this);
-    // postToFirebase({ id: "analytics", action: "zonedrawStart" });
   },
 
   disable: function () {
     if (!this._enabled) return;
     L.Handler.prototype.disable.call(this);
-    // postToFirebase({ id: "analytics", action: "zonedrawEnd" });
+    this.setZIndex();
+    window.map.fire("wasabee:op:change");
```

Comment:
is this only to update zone dialog ?

Reply:
yes, to make the stop signs go away

Reply:
I'd rather directly update the dialog instead of lying on the event
passing the dialog with the `options` and calling `this.parent.update()`

Reply:
or better still, call this.update() in the same method that disables the ZonedrawHandler.

Reply:
but there is `ZonedrawHandler._keyUpListener` ?

Reply:
ZonedrawHandler.disable() calls ZonedrawHandler.removeHooks()

Reply:
this doesn't solve the issue, you want to update the dialog on [ESC]

Reply:
oh, good catch.

Reply:
neither parent.update() nor this.parent.update() works, parent is "window".

calling the hook when [esc] is pressed

Reply:
ofc, you need to define `parent` at initialization using the unused `options`